test/ios/OrtExtensionsUsage/Podfile

```diff
@@ -5,7 +5,7 @@ target 'OrtExtensionsUsage' do
   use_frameworks!
 
   # Pods for OrtExtensionsUsage
-  pod 'onnxruntime-c'
+  pod 'onnxruntime-objc'
```

Comment:
Is there any version restriction here, something like onnxruntime-objc >= 1.16?

Reply:
yes, 1.16 is the minimum required version now. it should get the latest like this, but I added `>= 1.16.0`.